udc/src/test/java/io/crate/udc/ping/PingTaskTest.java

```diff
@@ -102,16 +102,16 @@ public void testIsEnterpriseField() throws Exception {
             "http://dummy",
             Settings.EMPTY
         );
-        assertThat(pingTask.isEnterprise(), is("false"));
+        assertThat(pingTask.isEnterprise(), is("true"));
 
         pingTask = new PingTask(
             clusterService,
             clusterIdService,
             extendedNodeInfo,
             "http://dummy",
-            Settings.builder().put(SharedSettings.ENTERPRISE_LICENSE_SETTING.getKey(), true).build()
+            Settings.builder().put(SharedSettings.ENTERPRISE_LICENSE_SETTING.getKey(), false).build()
```

Comment:
pls replace the value with a settting.getDefault() call, so this will never go out of sync in the future